firmware/esp32_ttgo/config.h

```diff
@@ -0,0 +1,54 @@
+
+// secrets in a separate file so we don't accidentally commit credentials
+#if __has_include("secrets.h")
+#include "secrets.h"
```

Comment:
I'm going to add an untracked `secrets.h` file to the repo, with those commented out lines and a note that this is where you should add secret information.